yml/cdb_latiss.yaml

```diff
@@ -921,6 +921,10 @@ tables:
     "@id": "#visit1_quicklook.n_inputs"
     datatype: int
     description: Number of CCDs used to compute the visit aggregates.
+  - name: pixel_scale
+    "@id": "#visit1_quicklook.pixel_scale"
+    datatype: float
+    description: Measured detector pixel scale (arcsec/pixel).
```

Comment:
https://github.com/lsst/sdm_schemas/pull/266/files#r1809659673

Reply:
So here is it:
`ivoa:unit: arcsec/pixel`
(as opposed to `fits:tunit: arcsec/pixel` above.  In truth, I have no idea what the criterion is for using one or the other, I'm just trying to follow the established pattern!)?

Reply:
> So here is it: `ivoa:unit: arcsec/pixel` (as opposed to `fits:tunit: arcsec/pixel` above. In truth, I have no idea what the criterion is for using one or the other, I'm just trying to follow the established pattern!)?

This is up to you really if you want to include these as units, but it is preferable to specify units using the appropriate fields if possible rather than putting this in the description.

Also, please use `ivoa:unit` rather than `fits:tunit`. At some point we'll be converting all the `fits:tunit` fields to `ivoa:unit`. (This is something someone did originally which was copied but we actually use IVOA units so it is more correct to use this field instead.)

Reply:
I used `ivoa:unit` for the `cdb_*.yaml` files, but `fits:tunit` for `apdb.yaml` since every other entry in that file uses it (we are trained for consistency within a file 😉).

Reply:
It's still on our list to unify the usage.  For now I agree with your approach of keeping it consistent within a given file.